src/utils/backend.js

```diff
@@ -108,7 +88,7 @@ export default class Backend {
     return backendFetch(`tips${query}`);
   };
 
-  static addToken = async (address) => backendFetchNoTimeout('tokenCache/addToken', {
+  static addToken = async (address) => backendFetch('tokenCache/addToken', {
```

Comment:
this request can take long, when it times out the user gets to see a non working version of word-bazaar, as his token is not indexed and cached properly yet

Reply:
actually, timeout in `backendFetch` was removed in 22f0ee47c8801f28c09cc839129f11ad2ad78f42